components/modal/Modal.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * Author: bladimirardiles
+ * Component File Name: Modal.js
+ * Component Name: Modal
+ * Project: design-system
+ * Date: Sun 10/12/2023 - 21:21
+ */
+
+import React from 'react';
+import Button from '../button/Button';
+import { IModalComponentProps } from './modal.interface';
+import { EIConTypes } from '../icon/icon.enum';
+import {
+  EButtonIconPosition,
+  EButtonSizes,
+  EButtonTypes,
+} from '../button/button.enum';
+import {
+  actions,
+  content,
+  header,
+  overlay,
+  title,
+  topContainer,
+} from './modeal-css';
```

Comment:
there is a typo in the name of the css file

Reply:
It is corrected. Thanks for noticing it.